scripts/cat/pelts.py

```diff
@@ -749,8 +748,9 @@ def check_and_convert(self, convert_dict):
         if self.white_patches in convert_dict["old_creamy_patches"]:
             self.white_patches = convert_dict["old_creamy_patches"][self.white_patches]
             self.white_patches_tint = "darkcream"
-        elif self.white_patches in ["SEPIAPOINT", "MINKPOINT", "SEALPOINT"]:
-            self.white_patches_tint = "none"
+
+        elif self.white_patches in ['SEPIAPOINT', 'MINKPOINT', 'SEALPOINT']:
```

Comment:
This is so hilariously minor but since I've asked for other changes, could these single quotes be replaced with double quotes?